doc/move_group_interface/move_group_interface_tutorial.rst

```diff
@@ -28,20 +29,19 @@ After a short moment, the RViz window should appear and look similar to the one
 
 Expected Output
 ---------------
-See the `YouTube video <https://youtu.be/_5siHkFQPBQ>`_ at the top of this tutorial for expected output. In RViz, we should be able to see the following:
- 1. The robot moves its arm to the pose goal to its front.
- 2. The robot moves its arm to the joint goal at its side.
- 3. The robot moves its arm back to a new pose goal while maintaining the end-effector level.
- 4. The robot moves its arm along the desired Cartesian path (a triangle down, right, up+left).
- 5. A box object is added into the environment to the right of the arm.
-    |B|
-
- 6. The robot moves its arm to the pose goal, avoiding collision with the box.
- 7. The object is attached to the wrist (its color will change to purple/orange/green).
- 8. The object is detached from the wrist (its color will change back to green).
- 9. The object is removed from the environment.
-
-.. |B| image:: ./move_group_interface_tutorial_robot_with_box.png
```

Comment:
why remove this image?

Reply:
The image was from the old tutorial and I removed it form the middle of the bulleted list because I found it intrusive. I decided not to replace it above or bellow because there is already an embedded video above.